docs/NMOS With H.264.md

```diff
@@ -58,11 +58,116 @@ The term 'strict-CBR' corresponds to the definition of constant bit-rate at sect
+  - "1b", "1.1", "1.2", "1.3"
+  - "2", "2.1", "2.2"
+  - "3", "3.1", "3.2"
+  - "4", "4.1", "4.2"
+  - "5", "5.1", "5.2"
+  - "6", "6.1", "6.2"
+
+Informative note: The names of the levels in string form have been derived from the names used at Annex A of the H.264 standard.
+
+The [NMOS Parameter Registers] is the official reference for the levels strings. In the case of discrepancies between this specification and the [NMOS Parameter Registers], the registers have precedence.
+
+The Flow's `profile` and `level` attributes map to the `profile-level-id` parameter of the SDP transport file. See section SDP format-specific parameters.
+
+The Flow's `profile` and `level` attributes map to the members profile_idc, level_idc and constraint_set?_flag of the AVC_video_descriptor of an MPEG2-TS transport stream. See section Multiplexed Flows.
+
+Informative note: The Flow's `profile` and `level` attributes are always required. The SDP transport file `profile-level-id` parameter MAY be omitted when matching the default value.
+
+- [Bit Rate](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#bit-rate)
+  The Flow resource MUST indicate the target encoding bit rate (kilobits/second) of the H.264 bitstream. It SHALL comply with the stream's active parameter sets. The `bit_rate` integer value is expressed in units of 1000 bits per second, rounding up.
+
+  Informative note: The H.264 bitstream is not required to transport HRD parameters such that an H.264 decoder may not know the actual target bitrate of a stream. There are bit rate limits imposed by the level of the coded bitstream. IS-11 may be used to constraint the Sender to a target bit rate compatible with the Receiver Capabilities.
+
+ - [Constant Bit Rate](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#constant-bit-rate)
+  The Flow resource MUST indicate if it operates in constant bit rate (CBR) mode or variable bit rate mode (VBR or other). When operating in constant bit rate mode the `bit_rate` corresponds to the constant encoding bit rate. Otherwise it corresponds to the maximum encoding bit rate. Since the default value of this attribute is `false`, a Flow MAY omit this attribute when using a variable bitrate mode.
+
+Informative note: The maximum bit rate information relates to the codec profile / level limits and the HRD buffering model. The CBR versus VBR mode of operation of the encoder provide essential clues about the coded bitstream produced.
+
+Informative note: For Flows compliant with ST 2110-22, the constant bit rate mode is more appropriately described as a strict-CBR mode where "The video compression or the packetization of the video compression shall produce a constant number of bytes per frame. The packetization shall produce a constant number of RTP packets per frame.". For other Flows, not compliant with ST 2110-22, it is the constant bit rate definition of of the H.264 specification that prevail.
```

Comment:
How does a Controller know if the Flow is ST 2110-22 compliant? I think maybe it's clear at the Sender and/or in the SDP file?

Reply:
Changed to "For streams compliant with ST 2110-22" ... We should revisit when doing the Sender attributes section. SSN still optional as per ST 2110-22 (2022 version)